lib/graphql/schema.graphql

```diff
@@ -5,6 +5,7 @@ type Query {
   physicians: [Physician!]
   applications: [Application!]
   permits: [Permit!]
+  applicant(id: ID!): Applicant
```

Comment:
Can we put this line after `applicants`? We want to group related APIs together in our GraphQL docs